deepchem/models/tests/test_ferminet.py

```diff
@@ -5,60 +5,36 @@
 import pytest
 import numpy as np
 try:
-    from deepchem.models.torch_models.ferminet import Ferminet
+    from deepchem.models.torch_models.ferminet import FerminetModel
+    # When pytest runs without pytorch in the environment (ex: as in tensorflow workflow),
+    # the above import raises a ModuleNotFoundError. It is safe to ignore it
+    # since the below tests only run in an environment with pytorch installed.
 except ModuleNotFoundError:
     pass
 
 
 @pytest.mark.torch
-def test_prepare_input_stream():
```

Comment:
Why are we deleting this older test?

Reply:
The old test checks for input streams shape, which is nothing but each electrons distance from each nuclei, this can be done using deepchem utils pairwise distance itself and doesn't require tests!